xonsh/aliases.py

```diff
@@ -73,6 +73,19 @@ def source_bash(args, stdin=None):
     return
 
 
+def exec(args, stdin=None):
```

Comment:
Okay, that makes sense. I'll change the name to xexec and make it a wrapper around execx.